db/schema.rb

```diff
@@ -53,11 +53,14 @@
     t.index ["slug"], name: "index_communities_on_slug", unique: true, using: :btree
   end
 
-  create_table "index_choices", force: :cascade do |t|
-    t.integer  "index_id"
-    t.integer  "choice_id"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+  create_table "indices", force: :cascade do |t|
+    t.datetime "finalized_at"
+    t.integer  "category_id"
```

Comment:
Correct, I guess I was asking 'Do we want to bother? Would it make it more clear what we're doing?' rather than what exactly would need to be done.